README.md

```diff
@@ -1,68 +1,45 @@
-# A TensorFlow Implementation of DC-TTS: yet another text-to-speech model
+# Testing STT performance on DC-TTS - _A Tensorflow Implementation of DC-TTS_.  
 
-I implement yet another text-to-speech model, dc-tts, introduced in [Efficiently Trainable Text-to-Speech System Based on Deep Convolutional Networks with Guided Attention](https://arxiv.org/abs/1710.08969). My goal, however, is not just replicating the paper. Rather, I'd like to gain insights about various sound projects.
+## Requirements:
 
-## Requirements
-  * NumPy >= 1.11.1
-  * TensorFlow >= 1.3 (Note that the API of `tf.contrib.layers.layer_norm` has changed since 1.3)
-  * librosa
-  * tqdm
-  * matplotlib
-  * scipy
+* NumPy >= 1.11.1
+* TensorFlow >= 1.3
+* Librosa
+* tqdm
+* matplotlib
+* scipy
 
 ## Data
 
-<img src="https://image.shutterstock.com/z/stock-vector-korean-alphabet-korean-hangul-pattern-693680611.jpg" height="200" align="right">
-<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/9c/Kate_Winslet_March_18%2C_2014_%28headshot%29.jpg/890px-Kate_Winslet_March_18%2C_2014_%28headshot%29.jpg" height="200" align="right">
-<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/Nick_Offerman_at_UMBC_%28cropped%29.jpg/440px-Nick_Offerman_at_UMBC_%28cropped%29.jpg" height="200" align="right">
-<img src="https://image.shutterstock.com/z/stock-vector-lj-letters-four-colors-in-abstract-background-logo-design-identity-in-circle-alphabet-letter-418687846.jpg" height="200" align="right">
+The original repository provides us with a [pre-trained model](https://www.dropbox.com/s/1oyipstjxh2n5wo/LJ_logdir.tar?dl=0) based on the LJ Speech Dataset. With that, we are able to test the performance of the TTS model with pre-defined or custom sentences (see further steps).

```

Comment:
Nice that you included links to the original repository since you overwrote the original readme. Good academic practice and nicely done.